### ISSUE TITLE
Can't run this example

### ISSUE THREAD

**Issue by xumx** [alexkuz/stellar-webpack#6]
I've tried running this visualization, but couldn't get it to work. 
I did `npm install`. followed by `npm run start`. But it fails with a whole lot of errors in the browser console.

I tried to update all packages and patch breaking changes in React/Webpack/Three. But still couldn't get it to run. 
WebGL seems to be running crazy.

Is it possible to patch this viz with a working version that I can test with `npm run start`?

**Comment by alexkuz** [comment]
yes, `react-three-renderer` lib (which is used here) relies heavily on React inner structure, so it doesn't work with the latest versions of React. To be fair, `stellar-webpack` also depends a bit on React internal stuff. I've updated dependencies and fixed errors, try it again.
